client/dashboard/domains/domain-connection-setup/components/support-link.tsx

```diff
@@ -0,0 +1,43 @@
+import { DomainConnectionSetupMode } from '@automattic/api-core';
+import { localizeUrl } from '@automattic/i18n-utils';
+import {
+	MAP_DOMAIN_CHANGE_NAME_SERVERS,
+	MAP_EXISTING_DOMAIN_UPDATE_A_RECORDS,
+} from '@automattic/urls';
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	Icon,
+} from '@wordpress/components';
+import { createElement, createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { help } from '@wordpress/icons';
+
+type SupportLinkProps = {
+	mode: DomainConnectionSetupMode;
+};
+export default function SupportLink( { mode }: SupportLinkProps ) {
```

Comment:
Would it be better to accept icon and message as props to make it more reusable? If so, we can also use it for the `SwitchSetup`. Besides, it doesn't seem to be "link". Instead, it's more like `help-message` 🤔